extension-manifest-v3/src/utils/api.js

```diff
@@ -9,71 +9,71 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 
-import * as cookies from './cookies.js';
+import jwtDecode from 'jwt-decode';
```

Comment:
is this needed? if we won't check the signatures decoding the JWT should be fairly simple